proxy.go

```diff
@@ -64,9 +87,10 @@ func getAllEnvVariables() {
 
 	// Get the AWS credentials
 	awsRegion = getEnvOrDefault("AWS_REGION", "eu-west-1", false)
-	awsBucket = getEnvOrDefault("AWS_BUCKET", "", true)
-	getEnvOrDefault("AWS_ACCESS_KEY_ID", "", true)
-	getEnvOrDefault("AWS_SECRET_ACCESS_KEY", "", true)
+	awsBucketPostfix = getEnvOrDefault("AWS_BUCKET_POSTFIX", "", true)
+	pathPrepend = getEnvOrDefault("PATH_PREPEND", "/", false)
+	// getEnvOrDefault("AWS_ACCESS_KEY_ID", "", true)
+	// getEnvOrDefault("AWS_SECRET_ACCESS_KEY", "", true)
```

Comment:
Maybe a comment here as to why you're leaving these checks for Future Jan? Or Just cut them out?

Reply:
Added in `25783a1`